src/orchestrator.ts

```diff
@@ -307,6 +308,38 @@ export class Orchestrator {
         );
     }
 
+    private callHttp(state: HistoryEvent[], req: DurableHttpRequest): Task {
```

Comment:
@cgillum - only change i might make is to this api shape? I think for node people may be more familiar with directly inputting the req object inputs

ex:
```
callHttp(state: HistoryEvent[], uri: string, method: string, content?: string | object) {
    if (content && typeof content !== 'string') {
        content = JSON.stringify(content);
    }
    const req = new DurableHttpRequest(method, uri, content);
    ...
}
```

Reply:
Yes, I love this suggestion! I didn't know how to do this in TypeScript, but I'll make this change now that you've shown it to me.